docs/user-guide/configure-certificates.md

```diff
@@ -1,70 +1,73 @@
 # Configuring Zowe certificates 
 
-As a system administrator, review this article to learn about the key concepts of Zowe certificates and the certificate configuring procudure.
+As a system administrator, review this article to learn about the key concepts of Zowe certificates, and options for certificate configuration. 
 
-Zowe uses a certificate to encrypt data for communication across secure sockets. The certificate used is specified in the `zowe.certificates` section of the `zowe.yaml`, and can either be a `PKCS12` certificate held in a USS keystore `.p12` file, or else a `JCERACKS` certificate held in a keyring.  
+Zowe uses a certificate to encrypt data for communication across secure sockets. The certificate used is specified in the `zowe.certificates` section of the `zowe.yaml` file. Certificates can either be a `PKCS12` certificate held in a USS keystore `.p12` file, or a `JCERACKS` certificate held in a key ring.  
```

Comment:
Is `JCERACKS` a valid designation for the JCERACFKS, JCECCARACFKS, etc.?

Reply:
> Is `JCERACKS` a valid designation for the JCERACFKS, JCECCARACFKS, etc.?

Typo, thanks for pointing out. Will fix in the latest commit.